tests/behat/features/templates/affinity_groups/affinity_groups-individual.feature

```diff
@@ -1,59 +1,165 @@
 @templates
 @api
 @javascript
-Feature: test individual Affinity Group page
-  TODO: verify this test checks the following
-    Affinity Group Logo Image
-    Affinity Group Name
-    Description
-    Join (if selected, button grays out and check mark appears next to “Leave”)
+
+Feature: Feature: test an Affinity Groups page
+  An Affinity Group page should display the title, logo, description, and tags.
+  Groups that have specified an optional Slack, Q&A, GitHub, or email link should
+  display a button with the appropriate link.
+  This test checks the following:
+    Join is & Leave an
     Join on Slack (if applicable, links to slack group)
     Visit Q&A Platform (if applicable, links to Ask.CI)
     Mailing List (if applicable, links to email)
     Github Organization (e.g. /affinity-groups/open-ondemand)
-    Coordinators (links to individual peoples profiles
+    Coordinators (links to individual peoples profiles)
     Events (if none, filler text “No upcoming events”)
-    Resources (links to individual resource pages)
-    People (individual AG member cards are shown)
+    Associated Resources accordion shows information about the resource on ACCESS RP AGs
+    Recommended Resources accordion shows recommendations on Community AGs
 
-  Scenario: Unauthenticated user Test the individual Affinity Group page
-    Given I am not logged in
-    When I go to "/affinity-groups/cloud-computing"
+  # TODO get this to work
+  # I have not be able to add an existing CI Link to the affinity group.
+  # The UI does not allow me to add an existing CI Link to the AG.
+  # The name of the CI Link gets appended with a variable suffix (like "ci-link-for-user-200 (5)")
+  # but I'm haven't figured out what that number should be ahead of time.
+  # Screenshot showing error on https://cyberteam-projects.slack.com/archives/C047PFB6UKE/p1694214896487699
+  #Scenario: Admin user adds a CI Link to the AG ACCESS Support
+    Given I am logged in as a user with the "administrator" role
+    # add a CI-Link to an AG
+    When I go to "node/327/edit"
+    Then I should see "Edit Affinity Group ACCESS Support"
+    # this CI link is created by amp_dev.install
+    When I fill in "Display CI Links on your Affinity Group" with "ci-link-for-user-200"
+    And I wait 1 seconds
+    When I press "Save"
+    And I wait 2 seconds
+    Then I should see "Affinity Group ACCESS Support has been updated."
+    When I follow "ci-link-for-user-200"
+    And I wait 1 seconds
+    Then I should see "ci-link-for-user-200"
 
```

Comment:
Note that the above scenario is commented out.  I was able to get this to work locally, but not on github for the reasons described in the comment.